src/pieces/messageCount.ts

```diff
@@ -112,7 +112,7 @@ async function sendLevelPing(msg: Message, user: SageUser): Promise<Message> {
 		.setDescription(embedText)
 		.addField('XP to next level:', user.levelExp.toString(), true)
 		.setColor(createLevelRgb(user.level))
-		.setFooter(`You can turn the messages off by sending \`${PREFIX}lp\``)
+		.setFooter(`You can turn the messages off by sending /togglelevelpings`)
```

Comment:
Since there's a difference between just sending a message and using a command, I'd suggest we move to this wording when we talk about using commands from now on.
```suggestion
		.setFooter('You can turn the messages off by using the `/togglelevelpings` command')
```